Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -262,7 +262,41 @@ internal virtual bool SendBytes(byte[] bytes, int channelId = Channels.DefaultRe
                 return false;
             }
 
-            return TransportSend(channelId, bytes);
+            // send to client or server
+            if (Transport.activeTransport.ClientConnected())
+            {
+                return Transport.activeTransport.ClientSend(channelId, bytes);
+            }
+            else if (Transport.activeTransport.ServerActive())
+            {
+                return Transport.activeTransport.ServerSend(connectionId, channelId, bytes);
+            }
+            return false;
+        }
+
+        // SendBytes to many. basically Transport.Send(connections) + checks.
+        internal static bool SendBytes(List<int> connectionIds, byte[] bytes, int channelId = Channels.DefaultReliable)
```

Comment:
You cannot call this method without allocating a byte[]

You cannot cache byte [] because they need the exact length.  A byte[] of size 100 will not work for a message of size 99.

If the goal is to do allocation free sends,  you must pass an arraysegment or a message

Reply:
ArraySegment would come in the next PR. the goal of this one was to just add Send(list) so we can still support telepathy in the alloc free send